src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { PeopleTable } from '../PeopleList';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then(peopleFromServer => {
+        const peopleWithParents = peopleFromServer.map(person => {
+          return {
+            ...person,
+            mother: peopleFromServer.find(personToFind => {
+              return person.motherName === personToFind.name;
+            }),
+            father: peopleFromServer.find(personToFind => {
+              return person.fatherName === personToFind.name;
+            }),
+          };
+        });
+
+        setPeople(peopleWithParents);
+      })
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) {
```

Comment:
You can move the loader to the container

```
<main className="section">
      <div className="container">
      <Loader />
      ...
```
      
Because now it appears on the page, but it is not visible, it is covered by navigation